addons/io_sketchfab_plugin/sketchfab/__init__.py

```diff
@@ -235,7 +235,10 @@ def clean_node_hierarchy(objects):
         root.select_set(True)
 
 class Cache:
-    SKETCHFAB_CACHE_FILE = os.path.join(os.path.dirname(__file__), ".cache")
+    SKETCHFAB_CACHE_FILE = os.path.join(
+        bpy.utils.user_resource("SCRIPTS", "presets", create=True),
```

Comment:
I 'd rather use something like "sketchfab" or anything specific to Sketchfab instead of "preset" as subfolder here, what do you think ?

Reply:
No strong opinion: depending on how the user installed the addon, there may or may not be an "addons/sketchfab-x-y-z" folder under bpy.utils.user_resource("SCRIPTS"), but looking for it seems overkill.

The term "presets" seems to be used for those kind of scenario, but I can rename it to something like "sketchfab_cache", might be easier to identify later for the user indeed.

Reply:
changed to sketchfab_cache